src/main/resources/db/migration/V1_16__DATA_MIGRATION.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE recommendations
```

Comment:
Please can we take the opportunity to bump the version of flyway to V10?

Reply:
done